filter/filter.go

```diff
@@ -31,13 +35,55 @@ func (f *Filter) IsExcluded(v string) bool {
 	return ok
 }
 
+// Validate validates that the data inside of the filters is right
+func (f *Filter) Validate() error {
+	// Validate that the Targets have the right format
+	if len(f.Targets) != 0 {
+		for _, t := range f.Targets {
+			split := strings.Split(t, ".")
```

Comment:
Don't think it make much difference performance wise, but would it be more logic to use [strings index](https://golang.org/pkg/strings/#Index) instead?